<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInputData">
      <attribute name="title">
       <string>Общие исходные данные</string>
      </attribute>
      <widget class="QLabel" name="labelPicture">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Picture</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableInput">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>341</width>
         <height>461</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>Расчетная длина (Lx), м</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Расчетная длина (Ly), м</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Расчетная длина (Lb), м</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Тип закрепления</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Расположение нагрузки</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Форма эпюры моментов</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelCode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Нормы</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxCode">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelElement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Элементы</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxElement">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelTypeSolve">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Тип расчета</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxTypeSolve">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelFormSection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Форма сечения</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxFormSection">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="boxSortament">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelSortament">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Сортамент</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxNumberSection">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelNumbeSection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>№ элемента</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxSteel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>280</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelSteel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Сталь</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCode_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Имя элемента</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTypeSection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Тип сечения</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boxTypeSection">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="textName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>320</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLoad">
      <attribute name="title">
       <string>Нагрузки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSpinBox" name="boxCountLoad">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableLoad">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>N, кг</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mx, кг*м</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>My, кг*м</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qx, кг</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qy, кг</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>P, кг</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>b, см</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOutputData">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Вывод по значениям</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableOutLoad"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGeneralOutputData">
      <attribute name="title">
       <string>Общий вывод</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableOutK"/>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="tableOutGeneral"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="buttonSolve">
     <property name="text">
      <string>Расчет</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QListWidget" name="listFiles">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="buttonFolder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelComment">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLineEdit" name="textFolder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>621</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>`</string>
     </property>
     <property name="cursorPosition">
      <number>1</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="buttonWord">
     <property name="text">
      <string>В Word</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
